.github/workflows/connect-dev-release-test.yml

```diff
@@ -39,9 +39,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          lfs: true
           submodules: recursive
 
+      # Pull only files needed for connect to save LFS bandwidth
+      - name: "Pull LFS files for connect"
+        run: git lfs pull --include "packages/connect-common/files/**/*"
```

Comment:
Is this even necessary? Are firmwares bundled into web builds or only to desktop builds? @mroz22

Reply:
@karliatto

Reply:
It seem's it's included in upload log from AWS so I guess it's needed.